content/version/2/0/code_of_conduct.fr.md

```diff
@@ -0,0 +1,132 @@
+# Charte Code de Conduite Contributeurs
+
+## Notre engagement
+
+En tant que membres, contributeurs et dirigeants, nous nous 
+engageons à rendre la participation dans notre communauté 

```

Comment:
J’aurais plutôt utilisé
```suggestion
engageons à faire que la participation dans notre communauté soit
```

Reply:
French version:
C'est un peu pareil à mes yeux, on ne perd le sens de la phrase, donc si les gens préfère, je pense que c'est une bonne idée !

English version:
It's kind of the same to me, you don't lose the meaning of the sentence, so if people prefer, I think it's a good idea!